rstudio-r4/tests/controls/user_spec.rb

```diff
@@ -0,0 +1,31 @@
+title 'RStudio User'
+
+control 'Common Users' do
+  impact 'high'
+  title 'The rstudio user should exist'
+  desc 'We create a new user on boot in start.sh. The default rstudio user should be 1000' \
+  'The UID 1001 should be free for us to create a user at run time'
+  tag 'user'
+  tag 'group'
+
+  describe user('rstudio') do
+    it { should exist }
+    its('uid') { should eq 1000 }
+  end
+
+  describe user(1001) do
+    it { should_not exist}
+  end
+end
+
+control 'Common Groups' do
+  impact 'high'
+  title 'The rstudio user should have the correct groups'
+  desc 'The rstudio user should have the group staff group to match Jupyter Notebook' \
+  'The users group is not currently added but it should, to match Jupyter.'
```

Comment:
Explanations for decisions like this are gold dust. Eventually they might more usefully be located with the code (Dockerfile in this case, I presume), rather than in the tests. I guess the reason you've put this here is to highlight differences between the r3 and r4 versions, which makes sense.